### DISCUSSION TITLE
Getting started section should prioritize mobile dapp instructions

### DISCUSSION THREAD

**Discussion by bogdan-marian** [WalletConnect/walletconnect-monorepo#2126] [Developer Support]
Getting started section should prioritize dapp section. This is especially important for mobile platforms (react-native, kotlin, swift)
I'm not able to find any instructions for v2 on getting started from client (dapp) perspective on these platforms: 
- react-native
- kotlin
- swift

**Comment by MinLofftP** [comment]
That can definitely be frustrating. I agree that it's important to prioritize the dapp section in the getting started guide, especially for mobile platforms. Have you tried reaching out to the community or checking out any online forums for help? Sometimes they can be a great resource for finding solutions to problems like this.

**Comment by finessevanes** [comment]
Hey - thank you for the feedback. Would you want to see something similar to [this](https://docs.walletconnect.com/2.0/)?
<img width="739" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/44731696/016a035f-f713-47bd-9763-6fc50eabe97f">


I believe this is what you might be looking for:
https://docs.walletconnect.com/2.0/android/sign/dapp-usage
https://docs.walletconnect.com/2.0/ios/sign/dapp-usage
